### DISCUSSION TITLE
How to hide `Uno.Extensions.ObjectExtensions` methods from Intellisense?

### DISCUSSION THREAD

**Discussion by xperiandri** [unoplatform/uno.extensions#2516] [Q&A]
These methods have riddled Intellisense. They annoyingly appear everywhere

**Comment by nickrandolph** [comment]
@xperiandri are you referring to the methods defined here https://github.com/unoplatform/uno/blob/1129f49e197bbd28a87991c04a08d3b2cac825e1/src/Uno.Foundation/Uno.Core.Extensions/Uno.Core.Extensions.Compatibility/Extensions/ObjectExtensions.cs#L23
These shouldn't be appearing as they're on an internal class.
Can you provide an example of the method that is appearing that shouldn't be?

**Reply by xperiandri** [reply]
![image](https://github.com/user-attachments/assets/1db7b121-7c26-4568-a124-177e03064388)
![image](https://github.com/user-attachments/assets/4df83b2c-068d-47d3-ba24-22a48d728b04)

**Reply by xperiandri** [reply]
What version was it fixed?

**Comment by nickrandolph** [comment]
It's these extension methods: https://github.com/unoplatform/uno/blob/ff2f3f26953c8edcb28b309238ba18b63202c8e1/src/Uno.Foundation/Uno.Core.Extensions/Uno.Core.Extensions.Compatibility/Extensions/ObjectExtensions.cs#L23
We'll work out how we can exclude these from the implicit usings

**Reply by xperiandri** [reply]
Thanks!
